src/styles/blocks/stars.scss

```
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
+    background-position: center;
+    margin-right: 4px;
+  }
+
+  & &__star:nth-child(-n+4) {
```

```suggestion
  & &__star:nth-child(-n+4) {
```
Reuse all styles from task with stars